spec/lib/reportsv2_spec.rb

```diff
@@ -237,7 +237,7 @@
       end
     end
 
-    context 'XLS reports' do
+    context 'XLS reports', :pro_account do
```

Comment:
The issue with `pry-byebug` dependency has been solved on master. Merging latest master to this branch makes all tests passing. I think @loadkpi now you can proceed.